.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+junit*.xml
```

Comment:
These changes mean that junit xml files will be saved to local machines when tests are run locally.